lib/carrierwave/mounter.rb

```diff
@@ -97,6 +98,15 @@ def remote_urls=(urls)
           uploader = blank_uploader
           uploader.download!(url, header || {})
           @uploaders << uploader
+        rescue CarrierWave::DownloadError => e
+          option_download_retry_count = option(:download_retry_count)
+          if option_download_retry_count && current_download_retry_count < option_download_retry_count
+            @current_download_retry_count += 1
+            sleep 5
+            retry
+          else
+            raise e
+          end
```

Comment:
Please do not expose a download-specific logic outside of the downloader.
This should be moved to somewhere in [CarrierWave::Downloader::Base](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/downloader/base.rb).

Reply:
Hi @mshibuya!
Thanks for review, and many thanks for carrierwave！
I fixed for move  to CarrierWave::Downloader::Base.

Reply:
@mshibuya 
Hi, our application require this option.
Is it possible to merge this PR?